duecredit/stub.py

```diff
@@ -52,7 +52,7 @@ def _donothing_func(*args, **kwargs):
 
 
 try:
-    from duecredit import due, BibTeX, Doi, Url, Text
+    from duecredit import due, BibTeX, Doi, Url, Text  # lgtm [py/unused-import]
```

Comment:
thank you. Could you please adjust while at it the above
```
Copyright:  2015-2019
... 
__version__ = '0.0.8'
```
with current year and beef version to 0.0.9 ? (can't suggest right here since not in a diff)

Reply:
Done.